src/components/VideoCall/Options.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext, useState } from 'react';
+import { SocketContext } from './VideoCall';
+
+function Options({ children }) {
+	const [idToCall, setIdToCall] = useState('');
+
+	const { me, callAccepted, name, setName, leaveCall, callUser, callEnded } =
+      useContext(SocketContext);
+
+	return (
+		<div>
+			<input onChange={(e) => setIdToCall(e.target.value)} type="text" />
+			{callAccepted && !callEnded ? (
+				<button onClick={leaveCall} type="button">Salir</button>
+			) : (
+				<button
+					onClick={() => {
```

Comment:
create the function, then pass to the component
avoid the arrow functions directly on the prop